aip_xx1_gen2_launch/config/lidar_gen2_1.yaml

```diff
@@ -0,0 +1,65 @@
+launches:
+  - sensor_type: hesai_OT128
+    namespace: top
+    parameters:
+      max_range: 300.0
+      sensor_frame: hesai_top
+      sensor_ip: 192.168.1.201
+      data_port: 2368
+      sync_angle: 180
+      cut_angle: 180.0
```

Comment:
Compared to [lidar_gen2.yaml](https://github.com/tier4/aip_launcher/blob/394db3d5e1d4f72ddfa4937206a44b57c6776d7d/aip_xx1_gen2_launch/config/lidar_gen2.yaml), only `sync_angle` and `cut_angle` are different